stock_picking_report_undelivered_product/models/stock_move.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    is_splitted = fields.Boolean(
```

Comment:
Why the need of this? I think you already have fields to know that something is a backorder.

Reply:
I want only display not partially backorder moves...
How can I do?

Reply:
I think too that this field is innecesary because maybe exists other way to get to obtain the undelivered products.

For example (related to stock_picking):

```
pending_picking_ids = self.env['stock.picking'].search(
            [
                ('group_id', '=', self.group_id.id),
                ('state', '!=', 'done'),
                ('picking_type_id.code', '=', 'outgoing')
            ]
        )
```

Reply:
The main problem adding also the new field is that all the old existing pickings won't display these undelivered products, which may be a big problem depending the DB. Search proposed by @victoralmau seems good, with a bit of fine-tuning (cancel ones shouldn't be included).

Reply:
Group_id??? I want to do pickings without sale orders..... So this search is not valid...
I agree with the problem of the old pickings... _Other options???

Reply:
What about accessing other pickings through `stock.picking`.`backorder_id` and `backorder_ids`?

Reply:
You also have `backorder_id` in `stock.move`

Reply:
Are you sure??
Why do you show me the way... ??? Please

Reply:
You are right, `group_id` not work because manual picking creation not relate this field; `backorder_id` not work in this use case.
Maybe the best option is similar to:

```
cancell_move_ids = self.env['stock.move'].search(
            [
                ('picking_id', '=', self.id),
                ('state', '=', 'cancel')
            ]
        )
```